.github/workflows/ci-terraform.yml

```diff
@@ -61,6 +62,7 @@ jobs:
         run: git diff --exit-code
 
   lint-find-dirs:
+    name: Find modules
     runs-on: ${{ fromJSON(inputs.runs-on) }}
     steps:
       - uses: actions/checkout@v4
```

Comment:
IMO our implementation is wrong. We should be linting what is modified by this PR.

Reply:
@osterman 
This is configurated here https://github.com/cloudposse/github-actions-workflows/blob/c7f6b19c202a372ad2edcb44b4ca92a8c3c720b1/.github/workflows/ci-terraform.yml#L114